include/swift/SIL/OwnershipUtils.h

```diff
@@ -64,26 +64,26 @@ struct ErrorBehaviorKind {
 /// so types/etc do not leak.
 struct OwnershipChecker {
   /// The list of regular users from the last run of the checker.
-  SmallVector<SILInstruction *, 16> RegularUsers;
+  SmallVector<SILInstruction *, 16> regularUsers;
 
   /// The list of regular users from the last run of the checker.
-  SmallVector<SILInstruction *, 16> LifetimeEndingUsers;
+  SmallVector<SILInstruction *, 16> lifetimeEndingUsers;
 
   /// The live blocks for the SILValue we processed. This can be used to
   /// determine if a block is in the "live" region of our SILInstruction.
-  SmallPtrSet<SILBasicBlock *, 32> LiveBlocks;
+  SmallPtrSet<SILBasicBlock *, 32> liveBlocks;
 
   /// The list of implicit regular users from the last run of the checker.
   ///
   /// This is used to encode end of scope like instructions.
-  SmallVector<SILInstruction *, 4> ImplicitRegularUsers;
+  SmallVector<SILInstruction *, 4> endScopeRegularUsers;
 
   /// The module that we are in.
-  SILModule &Mod;
+  SILModule &mod;
 
   /// A cache of dead-end basic blocks that we use to determine if we can
   /// ignore "leaks".
-  DeadEndBlocks &DEBlocks;
+  DeadEndBlocks &deadEndBlocks;
```

Comment:
We're generally favoring this convention for locals now, I think, but I don't think we've actually agreed to use it for fields.

Reply:
I thought that was the style that we were using in SILGen, no? I thought we were trying to standardize on SILGen style.

Maybe we should have a discussion about this on swift-dev and get something written up in a small style guide so there is an easy reference for the forgetful (i.e. me) among us. ; ).

Reply:
I just created a thread for this on swift-dev: https://forums.swift.org/t/style-used-in-sil-siloptimizer/16495. I don't want this to be held up by that conversation. I am happy to go back in afterwards and fix the casing if needed.